String+SFLoggingKit.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  String+SFLoggingKit.swift
+//  SFLoggingKit
+//
+//  Created by Aleksandar Sergeev Petrov on 23.11.21.
+//
+
+import Foundation
+
+/// String syntax sugar extension
+extension String {
+
+    var ns: NSString {
+        return self as NSString
+    }
+
+    var lastPathComponent: String {
+        return ns.lastPathComponent
+    }
+
+    var stringByDeletingPathExtension: String {
+        return ns.deletingPathExtension
```

Comment:
[optional]

Should we unify the use of implicit returns in this extension for the sake of consistency?